app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  def show
+    user_id = params[:id] || current_user.id
+    @user = User.find(user_id)
+  end
+
+  def item
```

Comment:
Yep! Just for the future, my comments always refer to the line above where the comment is.